src/main/java/nl/knaw/dans/dvcli/command/NotificationTruncate.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2024 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.dvcli.command;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import nl.knaw.dans.dvcli.action.BatchProcessor;
+import nl.knaw.dans.dvcli.action.ConsoleReport;
+import nl.knaw.dans.dvcli.action.Database;
+import nl.knaw.dans.dvcli.action.Pair;
+import nl.knaw.dans.dvcli.action.ThrowingFunction;
+import nl.knaw.dans.dvcli.config.DdDataverseDatabaseConfig;
+
+import picocli.CommandLine;
+
+import javax.validation.constraints.Positive;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@CommandLine.Command(name = "truncate-notifications",
+        mixinStandardHelpOptions = true,
+        description = "Remove user notifications but keep up to a specified amount.",
+        sortOptions = false)
+@Slf4j
+public class NotificationTruncate extends AbstractCmd {
+    protected Database db;
+    public NotificationTruncate(@NonNull Database database) {
+        this.db = database;
+    }
+
+    
+    @CommandLine.ArgGroup(exclusive = true, multiplicity = "1")
+    UserOptions users;
+
+    static class UserOptions {
+        @CommandLine.Option(names = { "--user" }, required = true, 
+                description = "The user whose notifications to truncate.")
+        int user; // a number, preventing accidental SQL injection
```

Comment:
The help does not tell it should be a number (positive or not-negative I suppose)